strawberry_django/mutations/fields.py

```diff
@@ -257,8 +259,9 @@ def create(self, data: dict[str, Any], *, info: Info):
 
 def get_pk(
     data: dict[str, Any],
+    key_attr: str,
 ) -> strawberry.ID | relay.GlobalID | Literal[UNSET] | None:  # type: ignore
-    pk = data.pop("id", UNSET)
+    pk = data.pop(key_attr, UNSET)
```

Comment:
Because I suggested changing `id` to `pk` in other places, I would keep this as `id`, because it is basically here for `GlobalID`s, and change the `"pk"` in the if below to `key_attr`

Reply:
Because key_attr is set above: `get_pk(vdata, key_attr=self.key_attr)` and key_attr has a default value of "pk", added fallback to id below. So we try to find data[key_attr] (which is pk by default) then id.